test/hot/subscribe.test.js

```diff
@@ -0,0 +1,111 @@
+'use strict';
+
+var test  = require('tape');
+var zuora  = require('../../zuora');
+var config = require('../../etc/config.test.json');
+var log    = require('../../lib/logger');
+
+log.level(log.bunyan.DEBUG);
+
+test.skip('subscribe: fail to create 1 account record', function (t) {
+  t.plan(9);
+
+  var fields = {
+    'Account' : {
+        'AccountNumber': '1234',
+        'AutoPay': false,
+        'Batch': 'this is gonna fail',
+        'BillCycleDay': 1,
+        'Currency': 'USD',
+        'Name': 'John "test" Smith',
+        'PaymentTerm': 'Due Upon Receipt',
+        'SubAccounts__c': '',
+        'UUIDLookUp__c': '',
+        'Status': 'Draft'
+    }
+  };
+
+  zuora.connect(config, function(err, z) {
+    t.ifError(err && err.message || err);
+    t.ok(z, 'expect valid \'z\' client');
+    if (err) return;
+    z.subscribe(fields, function(err, result) {
+      t.ifError(err && err.message || err);
+      log.debug(z.client._client.lastRequest);
+      log.debug(z.client._client.lastResponse);
+      log.debug(result);
+      t.ok(Array.isArray(result),           'result is an array');
+      t.equal(result.length, 1,             'result has an array length of 2');
```

Comment:
You want result to be an array of 2 elements, but you you are testing `t.equal(result.length, 1) :p